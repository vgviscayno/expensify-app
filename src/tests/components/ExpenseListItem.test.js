import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should render ExpenseListItem correctly', () => {
    const wrapper = shallow(
        <ExpenseListItem key={expenses[2].id} {...expenses[2]} />
    );
    expect(wrapper).toMatchSnapshot();
});
