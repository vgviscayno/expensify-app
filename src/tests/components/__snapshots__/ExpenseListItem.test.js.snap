// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/3"
  >
    <h3>
      Credit card
    </h3>
  </Link>
  <p>
    $
    45
     - 
    Jan 05 1970
  </p>
</div>
`;
